Source/WebCore/rendering/PathOperation.cpp

```diff
@@ -34,6 +34,22 @@
 
 namespace WebCore {
 
+const std::optional<Path> BoxPathOperation::getPath(const TransformOperationData& data) const
+{
+    if (auto d = data.motionPathData()) {
+        Path path;
+        auto rect = d->containingBlockBoundingRect;
+        auto shiftedPoint = d->offsetFromContainingBlock;
+        shiftedPoint.scale(-1);
+
+        rect.moveBy(shiftedPoint);
+        path.addRect(rect);
```

Comment:
I wonder if we can add functionality where we pass the border-radius of the element like Chrome does. There isn't any test for it, but I did see a demo where that works.

Reply:
(could be a follow up though)

Reply:
I think we are failing the first test because of this. I think its ok to add this as a follow up.